kythe/cxx/indexer/cxx/IndexerASTHooks.cc

```diff
@@ -1702,22 +1717,26 @@ bool IndexerASTVisitor::VisitDeclRefExpr(const clang::DeclRefExpr* DRE) {
 
 bool IndexerASTVisitor::VisitBuiltinTypeLoc(clang::BuiltinTypeLoc TL) {
   if (FLAGS_emit_anchors_on_builtins) {
-    return EmitTypeLocNodes(TL);
+    RecordTypeLocSpellingLocation(TL);
+    return true;
```

Comment:
```suggestion
```

Reply:
The linter doesn't care about PR titles, only commits.

Reply:
Also, the length requirements lead to less readable commit messages.

Reply:
I'm not claiming that the linter is *good*, just that it's *angery*.